File: leetcode-submissions/2938_Separate_Black_and_White_Balls.ts
function minimumSteps(s: string): number {
    let  counter=0;
        let answer=0;
        for(let  i=s.length-1 ;i>=0;i--){
            if(s.charAt(i)==="0"){
                counter++;
            }else{
                answer+=counter;
            }
        }
        return answer;
};

File: leetcode-submissions/2938_Separate_Black_and_White_Balls.cpp
class Solution {
public:
    long long minimumSteps(string s) {
        long long  counter=0;
        long long answer=0;
        for(int i=s.size()-1 ;i>=0;i--){
            if(s[i]=='0'){
                counter++;
            }else{
                answer+=counter;
            }
        }
        return answer;
    }
};